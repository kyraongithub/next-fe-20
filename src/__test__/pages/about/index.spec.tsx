import AboutPage from '@/pages/about';
import { render } from '@testing-library/react';

describe('About Page', () => {
  it('should render about page', () => {
    const { container } = render(<AboutPage />);
    expect(container).toMatchSnapshot();
  });
});
