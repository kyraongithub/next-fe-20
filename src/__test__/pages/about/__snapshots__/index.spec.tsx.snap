// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page should render about page 1`] = `
<div>
  <div>
    AboutPage
  </div>
</div>
`;
